src/glyph/mod.rs

```diff
@@ -258,55 +258,137 @@ impl Contour {
         self.points.first().map_or(true, |v| v.typ != PointType::Move)
     }
 
-    /// Converts the `Contour` to a [`kurbo::BezPath`].
+    /// Converts the `Contour` to a Vec of [`kurbo::PathEl`].
     #[cfg(feature = "kurbo")]
-    pub fn to_kurbo(&self) -> Result<kurbo::BezPath, Error> {
-        let mut path = kurbo::BezPath::new();
-        let mut offs = std::collections::VecDeque::new();
-        let mut points = if self.is_closed() {
-            // Add end-of-contour offcurves to queue
-            let rotate = self
-                .points
-                .iter()
-                .rev()
-                .position(|pt| pt.typ != PointType::OffCurve)
-                .map(|idx| self.points.len() - 1 - idx);
-            self.points.iter().cycle().skip(rotate.unwrap_or(0)).take(self.points.len() + 1)
-        } else {
-            self.points.iter().cycle().skip(0).take(self.points.len())
-        };
-        if let Some(start) = points.next() {
-            path.move_to(start.to_kurbo());
-        }
-        for pt in points {
-            let kurbo_point = pt.to_kurbo();
-            match pt.typ {
-                PointType::Move => path.move_to(kurbo_point),
-                PointType::Line => path.line_to(kurbo_point),
-                PointType::OffCurve => offs.push_back(kurbo_point),
-                PointType::Curve => {
-                    match offs.make_contiguous() {
-                        [] => return Err(Error::ConvertContour(ErrorKind::BadPoint)),
-                        [p1] => path.quad_to(*p1, kurbo_point),
-                        [p1, p2] => path.curve_to(*p1, *p2, kurbo_point),
-                        _ => return Err(Error::ConvertContour(ErrorKind::TooManyOffCurves)),
-                    };
-                    offs.clear();
+    pub fn to_kurbo(&self) -> Result<Vec<kurbo::PathEl>, Error> {
```

Comment:
I would have many of the same comments on this patch as I had on #116, it might be worth reading through those?